### DISCUSSION TITLE
BingMaps integration

### DISCUSSION THREAD

**Discussion by k4dmandu** [JaWeilBaum/pyqtlet2#29] [Q&A] [answered]
Is it possible to display bing maps as a layer? There is a plugin for leaflet due to the use of {q} instead of {xyz} in the service url, but how could it be integrated?

**Comment by JaWeilBaum** [comment, marked as answer]
Hey @k4dmandu you might be able to achieve this by adding the following [js plugin](https://github.com/digidem/leaflet-bing-layer) to the project - similar like I added the rotated marker to it. If you want you can try to solve this in a private fork and the create a merge request :)
